app/models/piece.rb

```diff
@@ -11,6 +11,15 @@ def user
     color == "white" ? game.white_player : game.black_player
   end
 
+  # Please note that this method does not test whether the move is valid
+  def capture_move?(new_rank, new_file)
+    if color == "white"
+      game.pieces.find_by(rank: new_rank, file: new_file, color: "black") ? true : false
+    else
+      game.pieces.find_by(rank: new_rank, file: new_file, color: "white") ? true : false
```

Comment:
I would assign a color variable in the if, then unify these two queries using the `color` var instead. This is good too, just duplicating some important code.